ingress-inquisition/main.go

```diff
@@ -4,29 +4,83 @@ import (
 	"context"
 	"fmt"
 	"log"
+	"os"
+	"text/tabwriter"
 
 	"github.com/aws/aws-sdk-go-v2/service/ec2"
 	"github.com/aws/aws-sdk-go-v2/service/securityhub"
-	awsauth "github.com/guardian/fsbp-tools/aws-common"
+	"github.com/guardian/fsbp-tools/common"
 	"github.com/guardian/fsbp-tools/ingress-inquisition/utils"
 )
 
+type SecurityGroupAndVpc struct {
+	SecurityGroup string
+	VpcName       string
+	VpcId         string
+}
+
+func getVpcDetails(ctx context.Context, ec2Client *ec2.Client, groupIds []string) []SecurityGroupAndVpc {
+	groupDescriptions, err := ec2Client.DescribeSecurityGroups(ctx, &ec2.DescribeSecurityGroupsInput{
+		GroupIds: groupIds,
+	})
+	if err != nil {
+		log.Fatalf("Error describing security groups: %v", err)
+	}
+
+	res := []SecurityGroupAndVpc{}
```

Comment:
This block of code is largely the same as it was in main, but instead of logging everything, we're building up a list of structs to log out a bit later.